bitnami/aspnet-core/templates/ingress.yaml

```diff
@@ -28,7 +27,7 @@ spec:
       http:
         paths:
           {{- if .Values.ingress.extraPaths }}
-          {{- include "common.tplvalues.render" (dict "value" .Values.ingress.extraPaths  "context" $) | nindent 10 }}
+          {{- toYaml .Values.ingress.extraPaths | nindent 10 }}
```

Comment:
Could you explain why we need to switch from `render` to `toYaml'?

Reply:
With the previous settings I got an helm template error, therefore I orientated on bitnami keycloak chart and taken over the same configuration. Any suggestion?

Reply:
Which is the error that appeared?

Reply:
'nil pointer evaluating interface {}' on extraPaths.

Reply:
This is basically the example from above, just need the ingress configuration with extraHosts